Sage/carbon
# chore(content): remove classic theme - FE-3105

### Proposed behaviour

- Remove redundant documentation
- Create an MDX example for Content
- Refactor stories file to be CSF
- Remove `classic` story

### Current behaviour

- Component still has our old style documentation
- No MDX example for Content
- Stories file is currently in `storiesOf` format
- `classic` story exists

### Checklist



### Additional context
N/A

### Testing instructions
- `git checkout remove_classic_theme_content`
- `npm start` to start Storybook
- Navigate to Content
- Example should be in MDX and have no `classic` themed example
